c2corg_common/document_types.py

```diff
@@ -7,3 +7,4 @@
 USERPROFILE_TYPE = 'u'
 WAYPOINT_TYPE = 'w'
 ARTICLE_TYPE = 'c'
+BOOK_TYPE = 'b'
```

Comment:
Could you also please add the `BOOK_TYPE` to this new constant: https://github.com/c2corg/v6_common/blob/b705a82ed66e806beeb1a59340a02d974f5e901b/c2corg_common/document_types.py#L11

Same for the reports.